### ISSUE TITLE
Please add bulk change option

### ISSUE THREAD

**Issue by innocent-velociraptor** [tappi287/openvr_fsr_app#52]
Hello,

It's more a new feature request than a bug report.
Would it be possible to add an option to be able to either bulk apply the same settings for all games in one's library or solely to manually selected games?

I think the idea is simliar to the one mentionned in the second part of issue #51.

**Comment by tappi287** [comment]
Hey, thanks, this is the right place for feature suggestions.

No, it is not possible, please see https://github.com/tappi287/openvr_fsr_app/issues/27 and https://github.com/tappi287/openvr_fsr_app/issues/6

**Comment by tappi287** [comment]
o_O do you have a collection of throw-away accounts? :] Greetings, guilty-snake

Yes, I need to avoid "suspicious" behaviour by the app. Eg. code for an completely automatic update is there but will even trigger Windows built-in malware protection on the system were I actually build the software.

**Comment by tappi287** [comment]
No, it's a not a problem when building but while running the code.

**Comment by tappi287** [comment]
If you would like to implement this please feel free to create a fork and I'll see where I can help.

To clear things up: 1. I do not want to potentially trigger AV from any software I release 2. I see no use in a bulk change of options: These plugins will worsen your image quality with no performance gain in 85% of use cases. They can however help in very special scenarios where a system is GPU limited with very high-res HMDs. So these need to be very finely tuned for the 2-3 apps where this might apply. Applying these plugins to a lot of apps simply does not make sense. If someone is just after sharpening filters: there are lot more configurable and performant options available e.g. [VRToolkit](https://vrtoolkit.retrolux.de)

I'll hope to get this clarification/explanation into the app at some point in the future. Currently my impression is user's send links to FSR/fixed FOV/OpenXR Toolkit and alike around as a generic one click approach to obtain better performance which simply isn't the case for any of those.

**Comment by tappi287** [comment]
I guess both of us lack the numbers to prove our points here :) I kindly ask to continue discussion about performance and hardware in forums/discord. This is not to say you're wrong and I'm right, just the wrong place for a discussion we could have for weeks without proper numbers showing CPU -and- GPU frame times with and without these plugins enabled on a broad selection of apps.

**Comment by tappi287** [comment]
Hmm, I can create an environment without problems using:
`npm v8.3.1`
`yarn 1.22.10`

Your npm install command installs the plugin into your global npm modules. Maybe try to uninstall and then install the exact version stated in the lock files: `4.5.15`, should be:
`yarn add @vue/babel-preset-app@4.5.15`
`yarn add @vue/cli-shared-utils@4.5.15`

and just in case anything else is missing:
`yarn add @babel/core@7.11.0 babel-loader@8.1.0 cache-loader@4.1.0 thread-loader@2.1.3 webpack@4.0.0`

**Comment by tappi287** [comment]
> Where should I run the commands from, if that matters?

Inside the directory where you cloned the git repo. `C:\Users\guilty-snake\Downloads\openvr_fsr_app-master`

> Windows CMD or node.js command prompt?

Does not matter as long as you are in the right dir.

> Isn't there something similar to Python, like pip install -r requirements.txt

That's what pipenv is for. Pip would use your system interpreter while pipenv creates a virtual environment for you.

> But vue CLI is still complaining:
Error: Cannot find module '@vue/cli-plugin-babel/preset'

There are plenty of suggestions you could try:
https://duckduckgo.com/?q=vue+cli-plugin-babel%2Fpreset
One suggestion is to install `@babel/core @babel/preset-env`but I can't try this here.

> And npm modules can be installed either globally or locally?
What's better in this case?

If you have only one node.js project it doesn't really matter but in general it will save you head aches if you place dependencies locally inside your project space.

**Comment by tappi287** [comment]
> I assumed pipenv created a virtual env just for python, but that for the rest of the libraries in other languages, it would be able to use those that are installed system wide. Overall, I'm quite confused.

pipenv is just for python. pipenv shell may changed your PATH sys variable and you were finally able to install the deps because yarn could not resolve to system wide deps.

> Do you think that would be possible to package a working building env inside a Docker container? Perhaps not, because it relies on Windows stuff?

Yes, the app is only useful on Windows(plugins are platform specific dll's) so a Docker container has no use here.

> No happy when running the tests though :(

Just run them a second time. Some files need to be created on first run.
